indico/web/client/js/types/urls.d.mts

```diff
@@ -0,0 +1 @@
+declare module 'indico-url:*';
```

Comment:
I'm curious, is there any way to give those imports at least basic typing (callable, taking a dict as its first argument etc)?

Reply:
Can't test it right now, but you could probably do something like this:

```ts
declare module 'indico-url:*' {
  export default function buildUrl(args: {[key: string]: any}): string;
}
```

Reply:
Seems to work:
![image](https://github.com/user-attachments/assets/80d1d69a-e20a-47cf-9ce8-76b664452ba0)